EventsExpress.Core/Services/AuthService.cs

```diff
@@ -125,22 +135,22 @@ public async Task<AuthenticateResponseModel> EmailConfirmAndAuthenticate(Guid au
             return new AuthenticateResponseModel(jwtToken, refreshToken.Token);
         }
 
-        public async Task ChangePasswordAsync(ClaimsPrincipal userClaims, string oldPassword, string newPassword)
+        public async Task ChangePasswordAsync(string oldPassword, string newPassword)
         {
-            var userDto = GetCurrentUser(userClaims);
+            var userDto = GetCurrentUser();
```

Comment:
I guess we do not need the whole User object here.
Account with its AuthLocal will be enough.